src/libs/actions/IOU.js

```diff
@@ -1079,6 +1082,14 @@ function getUpdateMoneyRequestParams(transactionID, transactionThreadReportID, t
         }
     }
 
+    // Add optimistic transaction violations if there is a policy
+    const currentTransactionViolations = allTransactionViolations[`${ONYXKEYS.COLLECTION.TRANSACTION_VIOLATIONS}${transactionID}`];
+    if (policy && policy.id) {
+        optimisticData.push(
+            ViolationsUtils.getViolationsOnyxData(updatedTransaction, currentTransactionViolations || [], policy.requiresTag, policyTags, policy.requiresCategory, policyCategories),
```

Comment:
Can we move the fallback ` || []` to the line where we define `currentTransactionViolations`?
```suggestion
    const currentTransactionViolations = allTransactionViolations[`${ONYXKEYS.COLLECTION.TRANSACTION_VIOLATIONS}${transactionID}`]  || [];
    if (policy && policy.id) {
        optimisticData.push(
            ViolationsUtils.getViolationsOnyxData(updatedTransaction, currentTransactionViolations, policy.requiresTag, policyTags, policy.requiresCategory, policyCategories),
```